src/components/calltree/CallTree.js

```diff
@@ -187,7 +187,7 @@ class CallTreeComponent extends PureComponent<Props> {
         onExpandedNodesChange={this._onExpandedCallNodesChange}
         selectedNodeId={selectedCallNodeIndex}
         expandedNodeIds={expandedCallNodeIndexes}
-        highlightString={searchString.toLowerCase()}
+        highlightRe={searchStringsRe}
```

Comment:
I generally prefer verbosity for something like this. I wouldn't know what this abbreviation is coming across it randomly.